boards/posix/native_posix/hw_models_top.c

```diff
@@ -134,13 +152,18 @@ void hwm_main_loop(void)
 		case IRQCNT:
 			hw_irq_ctrl_timer_triggered();
 			break;
+		case SDLEVENTTIMER:
+			break;
```

Comment:
If you create a new timer for the SDL library you should call the function to runs when the timer triggers from here. An empty case is not useful. The function you call from here should then update the timer trigger to some future value (or `NEVER`).

Reply:
Actually ```sdl_handle_synchronious_events``` is updating the timer and is putting it 10ms into the future after an event occurred